src/webpack-angular-types-plugin/plugin.ts

```diff
@@ -65,26 +79,27 @@ export class WebpackAngularTypesPlugin {
         module.addDependency(codeDocDependency);
     }
 
-    // noinspection JSMethodCanBeStatic
-    private collectModulesToProcess(
-        modules: Set<Module>,
-        tsProject: Project
-    ): [string, Module][] {
-        const modulesToProcess: [string, Module][] = [];
-        for (const module of modules) {
-            const filePath = module.nameForCondition();
+    private getModuleInfoIfProcessable(
+        module: Module
+    ): ModuleInformation | null {
+        const filePath = module.nameForCondition();
 
-            // Skip null values (e.g. raw files)
-            if (!filePath) {
-                continue;
-            }
+        // Skip null values (e.g. raw files)
+        if (!filePath) {
+            return null;
+        }
```

Comment:
Would it be a good idea to move this code to the filter statement in line 46? This could filter out pretty much everything unnecessary before even thinking about processing the modules